company_website/templates/estimate_project.haml

```diff
@@ -10,6 +10,12 @@
 
 {% block extra_head %}
     {% stylesheet 'estimate_project' %}
+    :javascript
+      function gtag_report_conversion(url) {
```

Comment:
Please also remember that in JS is convention to name methods/variables etc. with camelCase instead of snake_case

Reply:
Well apparently Google thinks otherwise, since this code comes entirely from their official instruction... 🤷

Reply:
Done. I've changed it, because it appears to be called only locally, via the onclick action of the tracked elements.
Unless Google likes to practice some obscure, cryptic bullsh*t like something you'd expect from Apple...
... do they? 😨

Reply:
![xd-696x456](https://user-images.githubusercontent.com/26469852/90410091-3e269700-e0aa-11ea-815c-8da26ea5cc3f.jpg)